### DISCUSSION TITLE
v1.3.x

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/nc-time-axis#61] [Releases]
The release manager for `v1.3.x` is @bjlittle 

The release feature branch is [v1.3.x](https://github.com/SciTools/nc-time-axis/tree/v1.3.x)

| Minor Release | Status | Milestone | Project | Date | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v1.3.0` | Completed | [v1.3.0](https://github.com/SciTools/nc-time-axis/milestone/1) | [v1.3.0](https://github.com/SciTools/nc-time-axis/projects/1) | 11 June 2021 | [v1.3.0](https://github.com/SciTools/nc-time-axis/releases/tag/v1.3.0) | [Available](https://anaconda.org/conda-forge/nc-time-axis) | [Available](https://pypi.org/project/nc-time-axis/1.3.0/) |

| Patch Release | Status | Milestone | Project | Date | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v1.3.1` | Completed | - | - | 14 June 2021 | [v1.3.1](https://github.com/SciTools/nc-time-axis/releases/tag/v1.3.1) | [Available](https://anaconda.org/conda-forge/nc-time-axis) | [Available](https://pypi.org/project/nc-time-axis/1.3.1/) |

**Comment by bjlittle** [comment]
I've tentatively put a release date of **30 July 2021** on the `1.3.0` release milestone, being none-the-wiser as to it's content. I'm not wedded to this date, it's just a speculative line in the sand. Happy to shift it to the left or right depending on the wisdom of the crowd and the release content we agree to deliver.

As a minimum, I suggest we at least:
- breath life back into `nc-time-axis`, and
- get it working the the latest `cftime` 1.5.x (at this time of writing).

We now have an associated `nc-time-axis` [milestone](https://github.com/SciTools/nc-time-axis/milestone/1) and [project](https://github.com/SciTools/nc-time-axis/projects/1) to manage and orchestrate the `1.3.0` release, as summarised in the above table.

Please add issues and pull-requests to both the milestone and the project to include them in the release, and then help where you can to get them across the line for the release.

Also, please assign yourself to an issue and/or a pull-request to own them. But what exactly does "own" mean?
- Assigning yourself to an **open issue** means that you will try to resolve the issue either by rejecting and closing it, or making sure the associated work is done "somehow". It doesn't necessarily mean that you'll do the work to address the issue (of course, you can if you want to :wink:), but it does mean is that you're happy to be the principle point-of-contact to the issue author. Also, other `nc-time-axis` core devs can sleep easy knowing that "you've got this"
- Assigning yourself to an **open pull-request** means that you intend to review and merge. Naturally, other members are free to review and comment on the pull-request, but you personally want to see the PR through to the end and bank it... and again, other `nc-time-axis` core devs can easily assess all [open pull-requests](https://github.com/SciTools/nc-time-axis/pulls) and quickly hone in on those that need attention.

If in doubt, just ask here or start a dedicated [GitHub Discussion](https://github.com/SciTools/nc-time-axis/discussions), or ping the troops with @SciTools/nc-time-axis-devs.

Let's see how we roll :+1:

**Comment by bjlittle** [comment]
Last call for changes to `nc-time-axis` release 1.3.0

I suggest that we take the opportunity to rename `master` to `main`, see this [discussion](https://github.com/SciTools/nc-time-axis/discussions/65), then cut the release after #46 is merged.

@SciTools/nc-time-axis-devs Thoughts?

**Reply by bjlittle** [reply]
Also, I'd be happy to go with a straight 1.3.0 rather than 1.3.0rc0.

If we've fluffed something up, then we can issue a quick patch. I genuinely think a release candidate is a wee bit too heavy handed for this package... but certainly for this release.

Happy to go with the crowd here... @SciTools/nc-time-axis-devs ?

**Reply by lbdreyer** [reply]
I agree with releasing 1.3.0 directly. I don't think a release candidate is required

**Reply by trexfeathers** [reply]
All sounds good to me
